MSFSTouchPortalPlugin/Services/PluginService.cs

```diff
@@ -375,6 +380,13 @@ internal class PluginService : IPluginService, IDisposable, ITouchPortalEventHan
       }
     }
 
+    private void UpdateSimConnectState() {
+      string stat = "true";
```

Comment:
I think you meant 'state'.

Reply:
Hmm, true... can't recall now  But it could also be short for "status" which maybe I was thinking of at the time?  :)